nodejs_container/native/src/config.rs

```diff
@@ -71,7 +71,7 @@ fn make_config(instance_data: Vec<InstanceData>) -> Configuration {
         bridges: Vec::new(),
         logger: LoggerConfiguration {
             logger_type: "debug".to_string(),
-            rules: LogRules::new(),
```

Comment:
In general I don't like this kind of thing slipped into an unrelated PR.  Feels like an amendment rider on a bill that must pass.  Not that I disagree with it, but it makes it hard to think about the PR in question...  I think we should just have a class of PR that's "Cleanup" or something.

Reply:
Yeah sorry @zippy, I think this actually reverts the reverse thing that slipped into a previous PR of mine, so it should be back to how it was before...

I guess I wasn't expecting this to get merged right away, anyway.